### PR TITLE
keygen file is taken from ssh output

### PR DESCRIPTION
**Pull request by marcin-usielski**

### CODE REVIEW

File: test/cmd/unix/test_cmd_ssh.py [hunk 1]
```diff
@@ -171,6 +173,88 @@ def test_ssh_change_prompt_in_the_same_line(buffer_connection, command_output_ch
...
+Please contact your system administrator.
+Add correct host key in /user/user2/.ssh/known_hosts to get rid of this message.
+Offending ECDSA key in /user/user2/.ssh/known_hosts:4
+Password authentication is disabled to avoid man-in-the-middle attacks.
+Keyboard-interactive authentication is disabled to avoid man-in-the-middle attacks.
+Agent forwarding is disabled to avoid man-in-the-middle attacks.
+X11 forwarding is disabled to avoid man-in-the-middle attacks.
+
+
+ute@192.168.255.126: Permission denied (publickey,password).
```
Comment (tkrol88) [inline test/cmd/unix/test_cmd_ssh.py:248]: please change IP address :)

File: test/cmd/unix/test_cmd_ssh.py [hunk 2]
```diff
@@ -171,6 +173,88 @@ def test_ssh_change_prompt_in_the_same_line(buffer_connection, command_output_ch
...
+
+
+root@host:~ > ssh-keygen -R 192.168.255.126 -f /user/user2/.ssh/known_hosts
```
Comment (tkrol88) [inline test/cmd/unix/test_cmd_ssh.py:251]: and here

File: test/cmd/unix/test_cmd_ssh.py (hunk 1 above)
Comment (marcin-usielski) [reply test/cmd/unix/test_cmd_ssh.py]: Done

File: test/cmd/unix/test_cmd_ssh.py (hunk 2 above)
Comment (marcin-usielski) [reply test/cmd/unix/test_cmd_ssh.py]: Done

**Pull request merged**